[2023-05-26T06:45:16.980Z] ReferenceError: categoryNo is not defined
    at updateCategory (C:\Users\Admin\Desktop\FoloosiBlog\src\controllers\v1\blogsCategoryController.js:96:30)
    at Layer.handle [as handle_request] (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\express\lib\router\layer.js:95:5)
    at next (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\express\lib\router\route.js:137:13)
    at authCheck (C:\Users\Admin\Desktop\FoloosiBlog\middlewares\auth.js:47:21)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[2023-05-26T07:06:51.649Z] Error: connect ECONNREFUSED ::1:8001
    at AxiosError.from (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:825:14)
    at RedirectableRequest.handleRequestError (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:2973:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at Socket.socketErrorListener (node:_http_client:502:9)
    at Socket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[2023-05-26T07:06:51.882Z] Error: connect ECONNREFUSED ::1:8001
    at AxiosError.from (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:825:14)
    at RedirectableRequest.handleRequestError (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:2973:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at Socket.socketErrorListener (node:_http_client:502:9)
    at Socket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[2023-05-26T07:24:33.589Z] Error: connect ECONNREFUSED ::1:8001
    at AxiosError.from (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:825:14)
    at RedirectableRequest.handleRequestError (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:2973:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at Socket.socketErrorListener (node:_http_client:502:9)
    at Socket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[2023-05-26T07:39:50.963Z] Error: connect ECONNREFUSED ::1:8001
    at AxiosError.from (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:825:14)
    at RedirectableRequest.handleRequestError (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:2973:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at Socket.socketErrorListener (node:_http_client:502:9)
    at Socket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[2023-05-26T07:48:58.523Z] ReferenceError: blogModel is not defined
    at getCategoryForFilter (C:\Users\Admin\Desktop\FoloosiBlog\src\controllers\v1\blogsCategoryController.js:351:36)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[2023-05-26T07:50:58.175Z] ReferenceError: blogModel is not defined
    at getCategoryForFilter (C:\Users\Admin\Desktop\FoloosiBlog\src\controllers\v1\blogsCategoryController.js:351:36)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[2023-05-26T07:51:00.925Z] ReferenceError: blogModel is not defined
    at getCategoryForFilter (C:\Users\Admin\Desktop\FoloosiBlog\src\controllers\v1\blogsCategoryController.js:351:36)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[2023-05-26T07:51:06.536Z] ReferenceError: blogModel is not defined
    at getCategoryForFilter (C:\Users\Admin\Desktop\FoloosiBlog\src\controllers\v1\blogsCategoryController.js:351:36)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[2023-05-26T07:51:50.720Z] ReferenceError: blogModel is not defined
    at getCategoryForFilter (C:\Users\Admin\Desktop\FoloosiBlog\src\controllers\v1\blogsCategoryController.js:351:36)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[2023-05-26T07:51:58.077Z] Error: connect ECONNREFUSED ::1:8001
    at AxiosError.from (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:825:14)
    at RedirectableRequest.handleRequestError (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:2973:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at Socket.socketErrorListener (node:_http_client:502:9)
    at Socket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[2023-05-26T07:52:03.374Z] ReferenceError: blogModel is not defined
    at getCategoryForFilter (C:\Users\Admin\Desktop\FoloosiBlog\src\controllers\v1\blogsCategoryController.js:351:36)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[2023-05-26T08:01:27.423Z] ReferenceError: firstData is not defined
    at getCategoryForFilter (C:\Users\Admin\Desktop\FoloosiBlog\src\controllers\v1\blogsCategoryController.js:350:38)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[2023-05-26T08:01:49.770Z] ReferenceError: firstData is not defined
    at getCategoryForFilter (C:\Users\Admin\Desktop\FoloosiBlog\src\controllers\v1\blogsCategoryController.js:350:38)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[2023-05-26T08:42:50.231Z] ReferenceError: firstData is not defined
    at getCategoryForFilter (C:\Users\Admin\Desktop\FoloosiBlog\src\controllers\v1\blogsCategoryController.js:350:38)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
[2023-05-26T08:47:29.792Z] Error: connect ECONNREFUSED ::1:8001
    at AxiosError.from (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:825:14)
    at RedirectableRequest.handleRequestError (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:2973:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (C:\Users\Admin\Desktop\FoloosiBlog\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at Socket.socketErrorListener (node:_http_client:502:9)
    at Socket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
[2023-06-02T05:41:30.881Z] Error: connect ECONNREFUSED ::1:8001
    at AxiosError.from (C:\Users\Admin\Desktop\node development\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:825:14)
    at RedirectableRequest.handleRequestError (C:\Users\Admin\Desktop\node development\FoloosiBlog\node_modules\axios\dist\node\axios.cjs:2973:25)
    at RedirectableRequest.emit (node:events:513:28)
    at eventHandlers.<computed> (C:\Users\Admin\Desktop\node development\FoloosiBlog\node_modules\follow-redirects\index.js:14:24)
    at ClientRequest.emit (node:events:513:28)
    at Socket.socketErrorListener (node:_http_client:502:9)
    at Socket.emit (node:events:513:28)
    at emitErrorNT (node:internal/streams/destroy:151:8)
    at emitErrorCloseNT (node:internal/streams/destroy:116:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
